### PR TITLE
Cleaned up repo structure

### PR DESCRIPTION
**Pull request by edmondop**

### CODE REVIEW

**Comment by holdenk** [comment]
I think the scripts need to be updated to reference the new dockerfile locations.

**Comment by edmondop** [comment]
> I think the scripts need to be updated to reference the new dockerfile locations.

yup sorry this was still draft

File: scripts/build.sh [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 source build_ray.sh
 # The raycluster operator doesn't handle upgrades well so delete + recreate instead.
 kubectl delete raycluster -n totallylegitco raycluster-kuberay || echo "No raycluster present"
-kubectl apply -f cluster.yaml
+kubectl apply -f k8s/cluster.yaml
 # Build the web app
 IMAGE=holdenk/fight-health-insurance:${FHI_VERSION}
-(docker pull "${IMAGE}" && sleep 10) || docker buildx build --platform=linux/amd64,linux/arm64 -t "${IMAGE}" . --push
-kubectl apply -f deploy.yaml
+(docker pull "${IMAGE}" && sleep 10) || docker buildx build --platform=linux/amd64,linux/arm64 -t "${IMAGE}" k8s --push
```
Comment (holdenk) [inline scripts/build.sh:23]: I don't think this will work, did you test the image build?

File: scripts/build_static.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+JS_PATH=fighthealthinsurance/static/js
+
+pushd "${JS_PATH}"
+npm ls >/dev/null 2>&1
+npm_dep_check=$?
+popd
+
+pushd "${JS_PATH}"
+if [ ${npm_dep_check} != 0 ]; then
+	npm i || echo "Can't install?" >/dev/stderr
```
Comment (holdenk) [inline scripts/build_static.sh:13]: I like this improvement :)

**Comment by edmondop** [comment]
Closing, superseded by #151

**Pull request closed without merging**